package.json

```diff
@@ -36,7 +35,7 @@
     "stats-webpack-plugin": "^0.2.1",
     "style-loader": "^0.13.1",
     "url-loader": "^0.5.7",
-    "webpack": "^1.9.11",
+    "webpack": "^2.5.1",
```

Comment:
@matanwerbner 2.6.1 is already the latest, can you revisit all the dependencies mentioned here?